bin/transync

```diff
@@ -46,8 +48,12 @@ try {
 
 var newDictionary = Object.assign({}, fromDictionary, toDictionary)
 var newContent = (path.extname(program.to) === '.json')
-  ? JSON.stringify(newDictionary, null, 2)
-  : yaml.safeDump(newDictionary)
+  ? JSON.stringify(newDictionary, null, codingStyle.indent_style === 'tab' ? "\t" : codingStyle.indent_size)
```

Comment:
Won't that render a `\t` instead of the tab character then? (that's the reason I always use double quotes when using `\t` or `\n`